core/src/main/java/com/linecorp/armeria/common/logging/BuiltInProperty.java

```diff
@@ -90,6 +90,10 @@
      * The port number is omitted when it is same with the default port number of the current {@link Scheme}.
      */
     REQ_AUTHORITY("req.authority"),
+    /**
+     * {@code "req.id"} - the id of the request.
```

Comment:
`{@code "req.id"} - the request ID.` ?

Reply:
Hmm.. I wrote this comment in terms of consistency.

I think it does not matter. If you want more it, I will edit it.
But I think current is okay to me.

Reply:
It has to be `ID` at least. I updated the file for you.